add speakers corner talk instructions (#198)
virtualscienceforum/virtualscienceforum: templates/talk_registered.md

We've added your talk to the schedule :tada:!

Below please find a checklist for your presentation.  
Please read it carefully and confirm that everything is clear.  
Good luck with your talk!

## Before your meeting

- We will send you an email containing the meeting key and your join URL in a moment.

>   **Confirm that you received the meeting key and URL by responding to this email.**

Author: This isn't an email, rather the bot response in the issue.

```suggestion
  **Confirm that you received the meeting key and URL by responding in this issue.**
```
